keps/sig-cli/kustomize-secret-generator-plugins.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Kustomize Secret Generator Plugins
+authors:
+  - "@sethpollack"
+owning-sig: sig-cli
+reviewers:
+  - "@monopole"
+  - "@Liujingfang1"
+approvers:
+  - "@monopole"
+  - "@Liujingfang1"
+editor: "@sethpollack"
+creation-date: 2019-02-04
+last-updated: 2019-02-04
+status: provisional
```

Comment:
Can you add the related SIGs section to the metadata? Some are listed as labels on the PR. Also, can you add SIG Apps.

Reply:
> Has this been discussed in SIG CLI? If not, I would suggest discussing it there as a next step.

Done on Feb 13th